src/main/java/use_case/CreateComment/CreateCommentInteractor.java

```diff
@@ -0,0 +1,55 @@
+package use_case.CreateComment;
+
+import entity.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+/**
+ * The Creation Comment Interactor.
+ */
+public class CreateCommentInteractor implements CreateCommentInputBoundary {
+    private final CreateCommentDataAccessInterface commentDataAccessObject;
+    private final CreateCommentOutputBoundary commentPresenter;
+    private final CommentFactory commentFactory;
+
+    public CreateCommentInteractor(CreateCommentDataAccessInterface commentDataAccessObject,
+                                   CreateCommentOutputBoundary commentPresenter,
+                                   CommentFactory commentFactory) {
+        this.commentDataAccessObject = commentDataAccessObject;
+        this.commentPresenter = commentPresenter;
+        this.commentFactory = commentFactory;
+    }
+
+    @Override
+    public void createComment(CreateCommentInputData createCommentInputData) {
```

Comment:
since we figured out the database, you can go ahead and implement a fallback here to prepare a fail view (i.e. comment creation unsuccessful due to input values being invalid). A comment's input value is invalid if it has no body.